File: Spring-backend/Springbackend/src/main/java/com/dravicenne/backend/models/Mail.java
package com.dravicenne.backend.models;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import java.io.Serializable;

@AllArgsConstructor
@NoArgsConstructor
@Data
@ToString
public class Mail implements Serializable {
    private String from;
    private String dest;
    private String body;
    private String topic;
}

File: Angular-frontend/src/app/Models/messagerie/mail/mail.ts
export class Mail {
  dest: string;
  from: string;
  topic: string;
  body: string;


  constructor(dest: string, from: string, topic: string, body: string) {
    this.dest = dest;
    this.from = from;
    this.topic = topic;
    this.body = body;
  }
}
